submit_and_compare/static/js/submit_and_compare_view.js

```diff
@@ -6,6 +6,7 @@ function SubmitAndCompareXBlockInitView(runtime, element) {
 	var publishUrl = runtime.handlerUrl(element, 'publish_event');
 
 	var $element = $(element);
```

Comment:
Oh, weird. It looks fine here, but is off on other views.
It looks like the original author (someone at OLI)
used hard-tabs there.

If @potsui wants to include it as a separate commit, that's fine.
If not, we can merge without though.